lib/Runtime/Library/JavascriptError.cpp

```diff
@@ -159,6 +159,54 @@ namespace Js
     }
 #endif
 
+    Var JavascriptAggregateError::NewAggregateErrorInstance(RecyclableObject* function, CallInfo callInfo, ...)
+    {
+        PROBE_STACK(function->GetScriptContext(), Js::Constants::MinStackDefault);
+        ARGUMENTS(args, callInfo);
+        ScriptContext* scriptContext = function->GetScriptContext();
+        JavascriptLibrary* library = scriptContext->GetLibrary();
+        JavascriptAggregateError* pError = scriptContext->GetLibrary()->CreateAggregateError();
+        Var newTarget = args.GetNewTarget();
+        RecyclableObject* errors = args.Info.Count > 1 ? JavascriptOperators::IterableToList(VarTo<RecyclableObject>(args[1]), scriptContext) : library->CreateArray(0);
+        Var message = args.Info.Count > 2 ? args[2] : library->GetUndefined();
+        pError->SetErrors(errors);
```

Comment:
Should make the errors a nullptr if length would be 0 and skip allocating the array - though will need to add a nullptr check when using it (though that should be added for safety anyway).